WeCantSpell.Hunspell/CandidateStack.cs

```diff
@@ -26,12 +26,17 @@ public static void Pop(ref CandidateStack stack)
     {
         if (stack._rest is { Count: > 0 })
         {
-            stack._rest.RemoveLast();
+            pop(stack._rest);
         }
         else
         {
             stack._s0 = null;
         }
+
+        static void pop(List<string> values)
```

Comment:
Some experimentation should be done with this. For example, is it better to just type the code out inline, or give it an inline attribute? Was it better in CollectionEx?

Reply:
I used a pattern match local variable instead.